src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,21 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @NotNull
+    @Email
     private String email;
+    @NotNull
+    @Size(min = 8, max = 256)
     private String password;
+    @NotNull
+    @Size(min = 8, max = 256)
+    private String repeateRassword;
```

Comment:
no need in not-null checks here, they are present in the custom validators

Reply:
took away redundant annotation